core/themes/basis/css/component/comment.css

```diff
@@ -163,3 +163,16 @@
 [dir="rtl"] .comment-footer {
   float: left;
 }
+
+a.comment-permalink  {
+  width: .7em;
+  height: .7em;
+  padding: .1em;
+  display: inline-block;
+  background-color: currentColor;
+  -webkit-mask-image: url("/core/modules/comment/images/link-icon.svg");
```

Comment:
` url("/core/modules/comment/images/link-icon.svg");`

If you define the url like that, it won't work on installs in subdirectories.